Dockerfile

```diff
@@ -41,9 +41,11 @@ RUN apk add --update --no-cache --virtual build-dependencies \
  rm -rf /usr/local/bundle/cache && \
  apk del build-dependencies
 
+ENV LD_PRELOAD="/usr/lib/libjemalloc.so.2"
```

Comment:
Looking at a fresh Rails app, the Dockerfile doesn't contain this line. Is this because we're using Ruby alpine image? This is a wild guess and I'm only asking for my own knowledge 🤷‍♂️

Reply:
Here: https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/docker-entrypoint.tt#L5

Reply:
Nice one thanks for this - it makes much more sense.